docs/ansible.posix.authorized_key_module.rst

```diff
@@ -240,6 +240,12 @@ Examples
         state: present
         key: https://github.com/charlie.keys
 
+    - name: Set authorized keys taken from url
+      ansible.posix.authorized_key:
+        user: charlie
+        state: present
+        key: "{{ lookup('url', 'https://github.com/charlie.keys', split_lines=False) }}"
+
```

Comment:
Hi! thank you for the pullrequest.

These sample task sections in the module document(`authorized_keys_module.rst`) are generated from the module code(`authorized_keys.py`) when a new version is released.
Therefore, is it possible to move this example task section to https://github.com/ansible-collections/ansible.posix/blob/main/plugins/modules/authorized_key.py#L84-L139 ?

Also, please note that you need to include the [changelog fragment file](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) as `trivial` in this pullrequest(. You can see a similar case in #233.

Thanks!

Reply:
Thank you. I've amended my commit.